srcpkgs/polybar/template

```diff
@@ -46,6 +46,11 @@ if [ "$build_option_network" ]; then
 	esac
 fi
 
+if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
+	makedepends+=" libatomic-devel"
+	configure_args+=" -DCMAKE_EXE_LINKER_FLAGS='-latomic'"
```

Comment:
Linker flags aren't the same as libraries the binary is linked against. If it works, I'd prefer to use `STANDARD_LIBRARIES`.

Reply:
Ok, I've pushed an update.